packages/v4/src/pages/home.js

```diff
@@ -0,0 +1,290 @@
+                    aria-label="view design guidelines"
+                    className="pf-u-pl-0 pf4-c-link__text ws-mdx-a"
+                  >
+                    <Link to="/design-guidelines/styles/colors">
+                      View design guidelines <ArrowRightIcon />
+                    </Link>
+                  </Text>
+                </div>
+                <div className="pf-u-pb-2xl">
+                  <Text component={TextVariants.h2}>Lay it out</Text>
+                  <Text component={TextVariants.p} className="ws-mdx-p">
+                    A layout provides a fully responsive page structure that keeps your components organized and aligned regardless of screen size.
+                  </Text>
+                  <Text
+                    aria-label="view layouts"
+                    className="pf-u-pl-0 pf4-c-link__text ws-mdx-a"
+                  >
+                    <Link to="/documentation/react/layouts/bullseye">
+                      View layouts <ArrowRightIcon />
+                    </Link>
+                  </Text>
+                </div>
+                <div className="pf-u-pb-2xl">
+                  <Text component={TextVariants.h2}>Build things your way</Text>
+                  <Text component={TextVariants.p} className="ws-mdx-p">
+                    Components are like building blocks. Designed to be flexible and modular, you can mix and match components to create a solution for almost any UI problem.
+                  </Text>
+                  <Text
+                    aria-label="view components"
+                    className="pf-u-pl-0 pf4-c-link__text ws-mdx-a"
+                  >
+                    <Link to="/documentation/react/components/aboutmodal">
+                      View components <ArrowRightIcon />
+                    </Link>
+                  </Text>
+                </div>
+                <div className="pf-u-pb-2xl">
+                  <Text component={TextVariants.h2}>See it in action</Text>
+                  <Text component={TextVariants.p} className="ws-mdx-p">
+                    Components are better together. Demos show how multiple components can be used in a single design. They also provide  some useful starter code, so the days of creating complex layouts from scratch are over.
+                  </Text>
+                  <Text
+                    aria-label="view demos"
+                    className="pf-u-pl-0 pf4-c-link__text ws-mdx-a"
+                  >
+                    <Link to="/documentation/react/demos/bulkselecttable">
+                      View demos <ArrowRightIcon />
+                    </Link>
+                  </Text>
+                </div>
+              </TextContent>
+            </GridItem>
+            <GridItem sm={12} md={6}>
+              <div type="image/svg+xml" className="pf4-c-image__laptop" aria-label="laptop image" />
+              <div type="image/svg+xml" className="pf4-c-image__phone" aria-label="phone image" />
```

Comment:
Why do we label these images? If they don't provide a purpose and are only presentational, we might be better off hiding them from screen reader users.

Reply:
Changed to `aria-hidden`.